nodes/nomos-node/config.yaml

```diff
@@ -70,22 +35,127 @@ blend:
       secs: 20
       nanos: 0
   membership:
-    - address: /ip4/127.0.0.1/udp/3001/quic-v1
-      public_key: [1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1]
-    - address: /ip4/127.0.0.1/udp/3002/quic-v1
-      public_key: [2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2]
-
+    - id: 12D3KooWKD62Q1nAAMHeV2Cx3rNNMzjgHkTT2edFMzHZ5ur7sGnh
+      address: /ip4/127.0.0.1/udp/8720/quic-v1
+      public_key: [223, 150, 178, 168, 141, 122, 185, 2, 106, 243, 47, 123, 25, 177, 191, 26, 
+                   98, 228, 30, 221, 115, 72, 32, 173, 97, 21, 71, 88, 227, 209, 172, 43]
+    - id: 12D3KooWA5LkQcuefjUBTktxCXUQ4rx4PcqRY1tgbSpV9eZDHWir
+      address: /ip4/127.0.0.1/udp/8721/quic-v1
+      public_key: [37, 199, 27, 35, 154, 26, 211, 224, 115, 140, 24, 212, 236, 242, 184, 68, 
+                   129, 92, 174, 95, 123, 211, 166, 115, 228, 128, 209, 15, 197, 113, 247, 31]
+da_network:
+  backend:
+    node_key: ea1e1dcc31612bd20987f017f0ca435cd2a59a4bd9fd6e14883b4803ae3b803d
+    membership:
+      assignations:
+      - - 12D3KooWA5LkQcuefjUBTktxCXUQ4rx4PcqRY1tgbSpV9eZDHWir
+        - 12D3KooWKD62Q1nAAMHeV2Cx3rNNMzjgHkTT2edFMzHZ5ur7sGnh
+      - - 12D3KooWKD62Q1nAAMHeV2Cx3rNNMzjgHkTT2edFMzHZ5ur7sGnh
+        - 12D3KooWA5LkQcuefjUBTktxCXUQ4rx4PcqRY1tgbSpV9eZDHWir
```

Comment:
This will probably break DA but the node will just not connect. Should be ok.

Reply:
This is actually a correct config, when two nodes are in both subnetworks.